libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -2339,6 +2370,15 @@ void AP_AHRS_NavEKF::request_yaw_reset(void)
     }
 }
 
+// set position source to either 0=primary or 1=secondary
+void AP_AHRS_NavEKF::set_position_source(uint8_t source_idx)
```

Comment:
I've renamed this method to set_posvelyaw_source to be consistent with the request to rename the equivalent methods in EKF3.  I haven't added the boolean return because for now at least it can only fail if the caller has provided an invalid source_idx (i.e. a value > 2).  This would be a programming error and is not possible to do from a lua script because the bindings enforce a value from 0 to 2.  The RC_Channel aux switch also only ever sends 0, 1 or 2.

Happy to add this in the future but for now I think it's ok to go without and avoid the additional code that would essentially just be checking for programming errors.